OfflineCI.ps1

```diff
@@ -0,0 +1,25 @@
+if (!($Host.Name -eq "Package Manager Host")){
+    echo "This script will only work when run from Package Manager Console"
+    echo "That is needed to correctly install NuGet packages"
+    exit 1
+}
+
+$CFSversion = "v0.2.5"
+if (!(Test-Path ".\CodeFileSanity.$CFSversion.exe") -or !(Get-Command ".\CodeFileSanity.*.exe" -ErrorAction SilentlyContinue)){
+    echo "Downloading CodeFileSanity"
+    Remove-Item * -Include CodeFileSanity.*.exe
+    wget "https://github.com/peppy/CodeFileSanity/releases/download/$CFSversion/CodeFileSanity.exe" -outfile "CodeFileSanity.$CFSversion.exe"
+}
+
+echo "Running CodeFileSanity"
+.\CodeFileSanity.*.exe
```

Comment:
would be nice to make these work under `mono` or `dotnet` for cross platform support (not required for this PR, just saying).